docs/introduction/core_use_cases/analytics.md

```diff
@@ -0,0 +1,187 @@
+---
```

Comment:
we can get rid of these yaml frontmatter sections, it's added [here](https://github.com/flyteorg/flyte/blob/master/docs/_ext/auto_examples.py#L69-L80) automatically at build time.